overview.md

```diff
@@ -0,0 +1,61 @@
+    - Frameworks limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
+- [Decentalized Voter Committees](addlink.com)
+    - Decentralized Voter Committees (DVCs) are a group of voters with aligned overall values and views. DVCs directly engage in the important high-level questions of Maker Governance.
+    - The objective of DVCs is to ensure that they represent MKR voters that are aligned with the values of that DVC. Members of DVCs will be compensated for their engagement under certain conditions, thereby incentivizing active governance. 
+- [Updated Tokenomics](tokenomics/launch-tokenomics.md)
+    - The Endgame Plan has two major tokenomics updates. First, annual MKR minting will be used to incubate SubDAOs, fund DVCs, and incentivize the workforce. These emissions counteract the MKR burn and reduce the likelihood of MKR becoming concentrated in the hands of large holders.
+    - The Endgame Plan introduces [Elixir](tokenomics/elixir.md) and [SubElixir](tokenomics/subelixir.md), which are liquidity pool tokens representing liquidity pools holding various combinations of MKR, ETH, DAI and SubDAO governance tokens. The aim of these tokens is twofold. It ensures alignment of interests between Maker Core and SubDAOs and also improves liquidity for all these tokens on the open market. 
+- Other Aspects
+    - [EtherDAI](tokenomics/etherdai.md): ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. EtherDAI is a synthetic Ether token backed by liquid staked Ether tokens and debt issued by Maker.
+    - In the final stages of Endgame, DAI is envisioned to turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
+    - MKR tokens will no longer function as a backstop in the event that the protocol is insolvent - instead, this will be reflected in the Target Price of DAI. 
+
+## Implementation
+
+### Phases
+The Endgame Plan will be implemented in four main phases. These are the Pregame, Early Game, Midgame, and Endgame phases. Due to the changes that are likely to occur to the Endgame Plan, this section only elaborates on details of the Pregame Phase. 
+
+The Pregame Phase will involve the following main changes, not necessarily in this order:
+- Publication of the Maker Constitution - a document that contains broad guidelines on the objectives and vision for Maker.
+- Launch of the first SubDAOs.
+- Updated tokenomics to include MKR emissions, Elixir, and SubElixir.
+- Reorganization of the workforce into entities that fit within the Endgame paradigm
+- Accumulating staked ETH and building on the staked ETH ecosystem through EtherDAI.
+
+### Stances
```

Comment:
Imo just leave this section out? I don't think it's important enough to be included in the overview. At least not as a separate heading.

Reply:
Agreed